src/rust.rs

```diff
@@ -34,12 +34,26 @@ use jsapi::{JSJitMethodCallArgs, JSJitGetterCallArgs, JSJitSetterCallArgs, CallA
 use jsapi::{NullHandleValue, UndefinedHandleValue, JSID_VOID};
 use jsapi::{CallArgsBase, CallReceiverBase, IncludeUsedRval, UsedRvalBase};
 use jsapi::CompartmentOptions;
+use jsapi::JS_DefineFunctions;
+use jsapi::JS_DefineProperties;
+use jsapi::JSFunctionSpec;
+use jsapi::JSNativeWrapper;
+use jsapi::JSPropertySpec;
+use jsapi::PropertyDefinitionBehavior;
 use jsval::UndefinedValue;
 use glue::{CreateAutoObjectVector, AppendToAutoObjectVector, DeleteAutoObjectVector};
 use glue::{NewCompileOptions, DeleteCompileOptions};
 use default_stacksize;
 use default_heapsize;
 
+fn ok(b: bool) -> Result<(), ()> {
```

Comment:
I'd prefer a to_result method defined on bool; I find it really easy to misread `ok(something)` as `Ok(something)`.